packages/react/src/analytics/integrations/GA4/validation/eventSchemas.ts

```diff
@@ -20,7 +20,6 @@ import {
   quantitySchema,
```

Comment:
This file contains a type error for the `signupNewsletterSchema`, where the `schema` parameter is not being typed (we have missed it). Can you fix that, please?